### PR TITLE
Project documentation (eduard.matei2905) 

### PR DESCRIPTION
**Pull request by lexofficial29**
Pull Request Overview

This pull request adds the documentation for the Obstacltron project.
TODO or Help Wanted

Please let me know if there is anything to change.
Author

Signed-off-by: Matei Eduard Gabriel [matei.eduard@proton.me](mailto:matei.eduard@proton.me)

### CODE REVIEW

File: docs/project/2024/eduard.matei2905/eduard.matei2905.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Description
+
+Obstacltron is an obstacle avoider robot built using Raspberry Pi Pico W. It utilizes an L298N motor driver, DC motors, and an HC-SR04 ultrasonic distance sensor.
+
+## Motivation
+
+The motivation behind this project was to create a simple yet effective robot that can navigate its environment autonomously by detecting and avoiding obstacles.
+
+## Architecture
```
[the remark this answers is not in the document]
Comment (lexofficial29) [reply docs/project/2024/eduard.matei2905/eduard.matei2905.md:16]: I added more information to the "Architecture" section and expanded a bit in the "Motivation" section too.

File: docs/project/2024/eduard.matei2905/eduard.matei2905.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+| HC-SR04 Ultrasonic Sensor       | Distance sensor    | [7 RON](https://www.optimusdigital.ro/en/ultrasonic-sensors/9-hc-sr04-ultrasonic-sensor.html) |
+| 2x 9V Battery                   | Power Source       | [5 RON](https://www.emag.ro/baterie-varta-superlife-zinc-carbon-9v-s1-4008496556632/pd/D0T5S3MBM/)
+
+## Software
+
+| Library             | Description             | Usage                                       |
+|---------------------|-------------------------|---------------------------------------------|
+
+
+## Links
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.matei2905/eduard.matei2905.md:66]: Please add some links related to your project?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/eduard.matei2905/eduard.matei2905.md (hunk 2 above)
Comment (lexofficial29) [reply docs/project/2024/eduard.matei2905/eduard.matei2905.md:66]: I added links for relevant components and tutorials used so far.

File: docs/project/2024/eduard.matei2905/eduard.matei2905.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# Obstacltron
+
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.matei2905/eduard.matei2905.md:2]: Please add a one line deacription of your project.

File: docs/project/2024/eduard.matei2905/eduard.matei2905.md
Comment (alexandruradovici) [inline docs/project/2024/eduard.matei2905/eduard.matei2905.md:1]: Rename the file to *index.md*.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/eduard.matei2905/eduard.matei2905.md
Comment (lexofficial29) [reply docs/project/2024/eduard.matei2905/eduard.matei2905.md:1]: I renamed the file and moved the description of the project to the top.

**Review by alexandruradovici — APPROVED** [review]

**Review by DanutAldea — APPROVED** [review]

**Pull request merged**